zoo-css-challenge/style.css

```
@@ -96,34 +106,73 @@ footer {
 
 #badge:hover {
   background-color: yellow;
+  transition-duration: 3s;
 }
 
 #badge p {
   text-align: center;
   font-size: 125%;
 }
 
+.image-container{
+  display: flex;
+  flex-direction: row;
+ 
+
+}
 .image-container img {
   vertical-align: top;
-  border: 5px solid #747c92;
+  border: 5px solid;
+ 
+
 }
 
-.image-container img:first-of-type {
+/*.image-container img:first-of-type {
   vertical-align: bottom;
+} #747c92;*/
+
+.image-container .img1{
+border-color: rgb(143, 202, 25);
 }
 
+.image-container .img2{
+  border-color: rgb(116, 7, 79);
+  }
+
+.image-container .img3{
+ border-color: rgb(233, 112, 13);
+  }
+
+
+
 .tigers {
   background-color: #0b3c49;
   color: #f0e100;
 }
 
+.tigers h2{
+color:rgb(205, 229, 252);
+}
+
 .giraffe {
-  background-color: #465775;
+  background-color: #483C46;
   color: #56e39f;
 }
 
-.giraffe h2 {
-  color: #ef6f6c;
+.giraffe h2, h3, p {
```

```suggestion
.giraffe h2, .giraffe h3, .giraffe p {
```
The `h3` and `p` selectors will apply to all `h3`s and `p`s on the webpage, but we only want them to apply to the giraffe section so we can update them to `.giraffe h3` and `.giraffe p` to only apply within the giraffe section